src/trimmer_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require_relative 'decorator'
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name[0, 10]
```

Comment:
This method ought to check if the output of @nameable.correct_name has a maximum of 10 characters. If it's longer it should trim the word😉. this has not correctly been implemented here👍🏼. Kindly do well to implement it properly👍🏼.

```suggestion
@nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name

```